reana/reana_dev/helm.py

```diff
@@ -0,0 +1,99 @@
+# -*- coding: utf-8 -*-
+#
+# This file is part of REANA.
+# Copyright (C) 2020 CERN.
+#
+# REANA is free software; you can redistribute it and/or modify it
+# under the terms of the MIT License; see LICENSE file for more details.
+
+"""`reana-dev`'s Helm commands."""
+
+import os
+import re
+import sys
+
+import click
+
+from reana.config import REPO_LIST_CLUSTER
+from reana.reana_dev.git import (
+    git_diff,
+    git_is_current_version_tagged,
+    git_push_to_origin,
+)
+from reana.reana_dev.utils import (
+    display_message,
+    get_component_version_as_semver2,
```

Comment:
Needs amending, as `get_component_version_as_semver2()` does not exist...

Reply:
... and need to import `get_docker_tag()`.  Otherwise it seems to work well :+1: